index.html

```diff
@@ -14,12 +14,14 @@
 </head>
 
 <body>
+    <div class="Headerback">
     <header class="header">
         <a class="logo" href="">My Logo</a>
         <nav>
             <img class="onion" src="image/Union.png" alt="" />
         </nav>
     </header>
+    </div>
```

Comment:
### Your Design
![header](https://user-images.githubusercontent.com/93039359/185756094-19a859ac-fe5d-42d9-b3b2-0a50ae8a0821.png)

### Figma Design
![Screenshot from 2022-08-20 21-04-13 (copy)](https://user-images.githubusercontent.com/93039359/185756100-65741881-7bf7-442b-b23f-6099d46e8aad.png)

- [ ] Red lines show the padding and margin. Please add it in your design to look follow the Figma template :blush: as requested by the previous reviewer :+1: